gtsam/geometry/triangulation.h

```diff
@@ -656,6 +659,10 @@ class TriangulationResult : public boost::optional<Point3> {
   bool outlier() const { return status == OUTLIER; }
   bool farPoint() const { return status == FAR_POINT; }
   bool behindCamera() const { return status == BEHIND_CAMERA; }
+  const gtsam::Point3& get() const {
```

Comment:
Why is that here? Seems to not have been needed before.

Reply:
It inherits from std::optional. boost::optional had a `get` call. Now we inherit from `std::optional`. So need to implement it